packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1034,7 +1034,10 @@ class CupertinoDialogAction extends StatelessWidget {
     this.isDestructiveAction = false,
     this.textStyle,
     @required this.child,
-  }) : assert(child != null);
+  }) : assert(child != null),
+       assert(isDefaultAction != null || isDestructiveAction != null),
```

Comment:
Is this condition correct? I don't think that a button needs to be default or destructive - consider an alert dialog with 5 buttons.

Reply:
It's just a check to prevent the user to explicitly set `null` the constructor, their default value is `false`.

Otherwise these conditions will throw an exception:

https://github.com/flutter/flutter/blob/cda5dfbf03ca6534b47ddd1b23c20b54a68a2fe1/packages/flutter/lib/src/cupertino/dialog.dart#L1146-L1152

Reply:
Ok, if the goal is to ensure that the values are non-null, then shouldn't we have 2 asserts? 1 for each property?

```dart
assert(isDefaultAction != null),
assert(isDestructiveAction != null),
// ...
```

Reply:
You're right! Fixed, thanks.